llvm/tools/sycl-post-link/SYCLDeviceLibReqMask.h

```diff
@@ -0,0 +1,50 @@
+//===----- SYCLDeviceLibReqMask.h - get SYCL devicelib required Info -----=-==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This pass goes through input module's function list to detect all SYCL
+// devicelib functions invoked. Each devicelib function invoked is included in
+// one 'fallback' SPIR-V library loaded by SYCL runtime. After scanning all
+// functions in input module, a mask telling which SPIR-V libraries are needed
+// by input module indeed will be returned. This mask will be saved and used by
+// SYCL runtime later.
+//===----------------------------------------------------------------------===//
+
+#pragma onece
+
+#include "llvm/ADT/Triple.h"
```

Comment:
It is better to move that include down to `.cpp` file

Reply:
Done.